Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```
@@ -273,7 +273,7 @@
         Amount: 1
         DefaultPrototype: Beaker
         ExamineName: Glass Beaker
-    componentRequirements:
+    materialRequirements:
```

yamllint (1.35.1)

[error] duplication of key "materialRequirements" in mapping

(key-duplicates)